src/components/App/App.tsx

```diff
@@ -0,0 +1,230 @@
+              </>
+            ),
+          },
+          {
+            title: (
+              <>
+                2.
+                <br /> Post-event support <br /> to ensure everything <br />{" "}
+                wraps up smoothly
+              </>
+            ),
+            description: (
+              <>
+                Our commitment doesn't end <br /> when the event does.
+              </>
+            ),
+          },
+          {
+            title: (
+              <>
+                3.
+                <br />
+                We tailor every event <br /> to your theme <br /> or request
+              </>
+            ),
+            description: (
+              <>
+                We pride ourselves on our individual <br /> approach to event
+                organizing
+              </>
+            ),
+          },
+        ]
+      : [
+          {
+            title: (
+              <>
+                1.
+                <br /> Planning the event, <br /> managing everything <br />
+                from logistics to decor
+              </>
+            ),
+            description: (
+              <>
+                to ensure a seamless <br /> and memorable experience.
+              </>
+            ),
+          },
+          {
+            title: (
+              <>
+                2.
+                <br /> Post-event support <br /> to ensure everything <br />{" "}
+                wraps up smoothly
+              </>
+            ),
+            description: (
+              <>
+                Our commitment doesn't end <br /> when the event does.
+              </>
+            ),
+          },
+          {
+            title: (
+              <>
+                3.
+                <br />
+                We tailor every event <br /> to your theme or <br /> request
+              </>
+            ),
+            description: (
+              <>
+                We pride ourselves on our individual <br /> approach to event
+                organizing
+              </>
+            ),
+          },
+        ];
+
+  return (
+    <div>
+      {/* Home Page List */}
+      <HorizontalList
+        containerClassName="bg-wheat"
+        unorderedListClassName="flex justify-between h-[260px] sm:flex-col sm:h-[100%] sm:flex-col sm:justify-center sm:items-center"
+      >
+        {HomePageItems.map((item) => (
+          <HorizontalList.Item
+            title={item.title}
+            description={item.description}
+            listClassName="flex flex-col w-[296px] 
+            text-center text-black tracking-[-0.02em] py-[24px] px-[24px] gap-[24px] sm:px-[12px] sm:gap-[12px]"
+            titleClassName="font-merriweather text-section-subtext text-lightGingerFlower leading-[38.4px]"
+            descriptionClassName="font-roboto text-[22px] text-black leading-[26.4px] sm:text-paragraph-sm"
+          />
+        ))}
+      </HorizontalList>
+      {/* Staffing Services List*/}
```

Comment:
The layout also breaks at 1357px for the other list. We might need a final adjustment to the Tailwind CSS media queries in tailwind.config.js to address this issue. <img width="1440" alt="Screenshot 2024-09-18 at 11 22 01 PM" src="https://github.com/user-attachments/assets/8ed521f6-f14b-43d0-9966-7fee1287516b">